src/music/dmusic.cpp

```diff
@@ -150,6 +317,20 @@ const char *MusicDriver_DMusic::Start(const char * const *parm)
 		return "AddPort failed";
 	}
 
+	IDirectMusicGraph *graph = NULL;
+	if (FAILED(proc.CoCreateInstance(
+		CLSID_DirectMusicGraph,
+		NULL,
+		CLSCTX_INPROC,
+		IID_IDirectMusicGraph,
+		(LPVOID*)&graph
+		))) {
```

Comment:
not entirely sure how OTTD does multi-line if statements, but pretty sure this isn't it :)